setup.py

```diff
@@ -112,7 +112,8 @@ def find_rel_test():
     zip_safe=False,
     install_requires=[
         'setuptools >= 0.6',
-        "vsc-base >= 2.4.18",
+        "vsc-base >= 2.5.0",
```

Comment:
(Kenneth Hoste, Fri, Mar 25, 2016 at 03:41:57PM -0700:)
> > @@ -112,7 +112,8 @@ def find_rel_test():
> >      zip_safe=False,
> >      install_requires=[
> >          'setuptools >= 0.6',
> > -        "vsc-base >= 2.4.18",
> > +        "vsc-base >= 2.5.0",
>
> what you specify here depends on how you bump the `vsc-base` version
> in https://github.com/hpcugent/easybuild-framework/pull/225

yes, of course they should be synchronized; see comments on that PR for
the versioning debate (short version: I don't care, just tell me the number)